precise_driver/src/precise_hw_interface.cpp

```diff
@@ -123,8 +133,20 @@ namespace precise_driver
             enableWrite(false);
             if(device_->init(profile_no_, profile_))
             {
-                if( device_->home())
+                if(device_->home())
                 {
+                    if(post_init_configuration_.size() > 0)
+                    {
+                        bool ret;
+                        ret = device_->moveJointPosition(profile_no_, post_init_configuration_);
+                        ret &= device_->waitForEom();
+                        if(!ret)
+                        {
+                            std::string msg = "Failed to move to post init configuration";
+                            ROS_ERROR_STREAM_NAMED("precise_hw_interface",msg);
+                        }
+                    }
+
                     device_->startCommandThread();
                     std::string msg = "successfully initialized";
```

Comment:
you could add some info to this string so we know in the res.message whether the post_init movement worked, e.g.
- `successfully initialized` when no post_init_config is set
- `successfully initialized and moved to post_init_configuration` when init and movement successful
- `successfully initialized but failed to move to post_init_configuration` when init worked, but movement failed